src/components/PropertyCard/property-card.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+import { ReactElement } from 'react';
+import { TProperties } from './property-card.types';
+
+const CardDiv = styled.div`
```

Comment:
Any chance we could stick these styles in the styled.ts file (you can bin what's already in there I think) 👍